src/api-reference/user-provisioning/spend/v4.spend-user-provisioning.md

```diff
@@ -750,4 +764,145 @@ Delete unused headers
 
 ## <a name="schema"></a>Schema
 
-`TODO`
+### <a name="spend-user-schema"></a>Spend User
+
+| Name                     | Type                                      | Format         | Description                                                                                                                             |
+| ------------------------ | ----------------------------------------- | -------------- | --------------------------------------------------------------------------------------------------------------------------------------- |
+| `reimbursementCurrency`  | `string`                                  | -              | **Required** Valid three digit currency code in the list of system reimbursement currencies.                                            |
+| `reimbursementType`      | `object`                                  | -              | **Required** The reimbursement type for the user. Supported values: `ACCOUNTS_PAYABLE`, `ADP_PAYROLL`, `CONCUR_PAY`, `PAY_PAL`, `OTHER` |
```

Comment:
This isn't a String?

Reply:
It is... maybe it is because its an enum within the system?